testing/bdd/features/styles/style_attribute_inherited.feature

```diff
@@ -42,3 +42,14 @@ Feature: Compute style attribute on a single EBU-TT Live element
     |           |          |                |          | tts:wrapOption       | span1   | wrap           |
 
 
+  Scenario: Circular style references should fail
+    Given an xml file <xml_file>
+    When it has a cell resolution of <cell_resolution>
```

Comment:
I put this test in this file because checking for circular references isn't really part of EBU-TT-3 > EBU-TT-D conversion, and this seemed like a sensible feature file to add to.
The reason why I've added this line (and the one below it) is because Cell Resolution and Extent are specified in the Examples table right at the top, and _would you believe_ pytest-bdd fails if you define a parameter in a test that isn't referenced.
I agree that these parameters are irrelevant to my scenario, so I'll have a go at refactoring this test.

Reply:
Thanks, pytest-bdd keeping us all honest there :-)